core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -592,10 +592,17 @@ object Fiber extends FiberPlatformSpecific {
    *
    * TODO: Switch to "streaming lazy" version.
    */
-  val dump: UIO[Iterable[Dump]] = UIO.effectSuspendTotal {
+  val dump: UIO[Iterable[Dump]] =
+    dumpAll(_rootFibers.asScala: @silent("JavaConverters"))
+
+  /**
+   * Collects a complete dump of the specified fibers and all children of the
+   * fibers.
+   */
+  def dumpAll(fibers: => Iterable[Fiber.Runtime[_, _]]): UIO[Iterable[Dump]] = UIO.effectSuspendTotal {
```

Comment:
Maybe the old one should be called `dumpAll`, and this one, `dumpOnly` or just `dump`?

Reply:
Yes. I like `dumpAll` for dump everything from the root fibers and just `dump` for this.

Reply:
Updated now.